packages/prediction-app/src/features/settings/ChapSettings/ChapSettings.module.css

```diff
@@ -0,0 +1,74 @@
+.container {
+    padding: 24px;
+}
+
+.header {
+    margin-bottom: 24px;
```

Comment:
May want to use logical css-properties, like `margin-block-end` for good measure. But nitpicking for now